src/Uno.UWP/Helpers/AndroidResourceNameEncoder.cs

```diff
@@ -31,6 +31,16 @@ public static string Encode(string key)
 				key = $"{NumberPrefix}{key}";
 			}
 
+			if (key.EndsWith(".9"))
+			{
+				// Specific handling of 9-patch extension
+				key = key.Substring(0, key.Length - 2).Replace(".", "_") + ".9";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'key.Substring(0, key.Length - 2).Replace' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5653558)